lib/parse/block-element.js

```diff
@@ -13,20 +13,29 @@ const TEXT_ELEMENTS = {
   blockquote: 'blockquote'
 };
 
+const isPullQuote = (elm) => {
+  return elm.classList.contains('q');
+};
+
 // inject parse to avoid recursive requires
 export default (parse, text) => {
   return (elm, textOpts) => {
     const tagName = elm.tagName.toLowerCase();
 
     if (BLOCK_ELEMENTS.has(tagName)) {
+      const type = TEXT_ELEMENTS[tagName] || 'block';
       const blockElement = {
-        type: TEXT_ELEMENTS[tagName] || 'block',
+        type,
         children: []
       };
       if (elm.childNodes.length) {
         parse(elm.childNodes, text(textOpts, elm), blockElement.children);
       }
 
+      if (type === 'blockquote') {
+        blockElement.pullQuote = isPullQuote(elm);
```

Comment:
@kesla like this: https://github.com/micnews/html-to-article-json/pull/100/commits/d99736f34d2001921c487fd987ae55e3b6c05a29 ?